moto/cloudformation/parsing.py

```diff
@@ -41,6 +41,7 @@
 from moto.sns import models as sns_models  # noqa
 from moto.sqs import models as sqs_models  # noqa
 from moto.stepfunctions import models as stepfunctions_models  # noqa
+from moto.ssm import models as ssm_models, ssm_backend  # noqa
```

Comment:
Just realized that the `ssm_backend` specifically refers to the backend for us-east-1, so it's not going to work for other regions. 
Out the top of my head, you might need to use `ssm_models.ssm_backends[region]`

Reply:
I updated the stack to be in `us-east-1`. Reckon that would work? c90cc01

Reply:
For our test, yeah, but moto users would also be stuck using us-east-1

Reply:
@bblommers Everything looks good now?